### PR TITLE
add jupyter for conversation

### PR DESCRIPTION
**Pull request by qbc2016**
As the title says.

### CODE REVIEW

File: notebook/conversation.ipynb [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25d30c19de76e93f",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "In this notebook, we will show a demo of how to program a multi-agent conversation in AgentScope.Complete code is in `examples/conversation/conversation.py`, which sets up a user agent and an assistant agent to have a conversation. When user input \"exit\", the conversation ends. You can modify the sys_prompt to change the role of assistant agent."
```
Comment (DavdGao) [inline notebook/conversation.ipynb:18]: Add space before "Complete"

File: notebook/conversation.ipynb [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6dbd5990-ed46-4727-84b1-d4131c5a6245",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install AgentScope"
```
Comment (DavdGao) [inline notebook/conversation.ipynb:36]: Follow README.md to install AgentScope.

File: notebook/conversation.ipynb [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c60227f7-7380-4f72-b653-bc7394cfaace",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import agentscope\n",
+    "from agentscope.agents import DialogAgent\n",
+    "from agentscope.agents.user_agent import UserAgent\n",
```
Comment (DavdGao) [inline notebook/conversation.ipynb:56]: should be `from agentscope.agents import UserAgent`

File: notebook/conversation.ipynb [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6057d6c5-a332-4949-84a7-6fbeda021561",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = None\n",
+    "while x is None or x.content != \"exit\":\n",
```
Comment (DavdGao) [inline notebook/conversation.ipynb:121]: Pls provide users a learning process here with two versions. The first version is without pipeline, and the second is with pipeline.

Comment (DavdGao) [reply notebook/conversation.ipynb:121]: E.g. to code a conversation easier, you can use pipeline in `agentscope.pipelines` and `agentscope.pipelines.functional`

**Review by DavdGao — COMMENTED** [review]
Please see inline comments

File: notebook/conversation.ipynb (hunk 2 above)
Comment (qbc2016) [reply notebook/conversation.ipynb:36]: Do you mean just write "Follow README.md to install AgentScope." or copy the codes from README.md here?

**Review by pan-x-c — APPROVED** [review]

**Review by DavdGao — APPROVED** [review]
LGTM

**Pull request merged**